src/main/java/lu/oop/server/api/controllers/UserController.java

```diff
@@ -102,46 +106,37 @@ public ResponseEntity<ComplaintModel> activeComplaint(@PathVariable Long id) {
         return ResponseEntity.ok(userService.getActiveComplaint(id));
     }
 
-    @GetMapping("{id}/friendships")
```

Comment:
It seems, that accepting this pull request will result in deletion of friendships under user controller